boards/st/b_u585i_iot02a/board.cmake

```diff
@@ -13,11 +13,9 @@ if(CONFIG_BUILD_WITH_TFM)
 endif()
 
 # keep first
-if(CONFIG_STM32_MEMMAP)
 board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
```

Comment:
Intentionally removing `--ersase` directive?
If so, could you mention it in the commit message?

Reply:
not necessarily, restored